hw/top_darjeeling/data/top_darjeeling.hjson

```diff
@@ -16,7 +16,7 @@
   datawidth: "32",
 
   // Enable RACL on Darjeeling based on the following configuration
-  racl_config: 'top_darjeeling/data/racl/racl.hjson'
+  racl_config: 'hw/top_darjeeling/data/racl/racl.hjson'
```

Comment:
Since these racl configs seem to be meant to go alongside the top hjson, would it make sense to do paths relative to this file? Relative paths from this repo's root won't work if the top hjson is in an out-of-tree repo.

The xbar configs work in this way (with a side of assumed file names based on the crossbar name).

Reply:
Fair point. I just updated that.